README.md

```diff
@@ -97,9 +97,41 @@ If you found any of the above modules useful, we would really appreciate if you
 }
 ```
 
+- [4] A. Rosinol, A. Gupta, M. Abate, J. Shi, L. Carlone. [**Kimera: from SLAM to Spatial Perception with 3D Dynamic Scene Graphs**](https://arxiv.org/abs/2101.06894). [arXiv:2101.06894](https://arxiv.org/abs/2101.06894).
+
+```bibtex
+@InProceedings{Rosinol21arxiv-Kimera,
+  title = {{K}imera: from {SLAM} to Spatial Perception with {3D} Dynamic Scene Graphs},
+  author = {A. Rosinol, A. Violette, M. Abate, N. Hughes, Y. Chang, J. Shi, A. Gupta, L. Carlone},
+  year = {2021},
+  booktitle = {arxiv},
+  pdf = {https://arxiv.org/pdf/2101.06894.pdf}
+}
+```
+
+## Open-Source Datasets
+
+We use a photo-realistic Unity-based simulator to test Kimera.
```

Comment:
this seems to hint that Kimera is only tested in simulation, which is not true. Maybe rephrase?

Reply:
```suggestion
In addition to the [real-life tests](http://ci-sparklab.mit.edu:8080/job/MIT-SPARK-Kimera/job/master/VIO_20Euroc_20Performance_20Report/) on the [Euroc](https://projects.asl.ethz.ch/datasets/doku.php?id=kmavvisualinertialdatasets) dataset, we use a photo-realistic Unity-based simulator to test Kimera.
```